docs/src/documentation/03-components/01-action/button/01-guidelines.mdx

```diff
@@ -36,7 +36,9 @@ check out our [interactive guide on action components](/guides/using-button-type
 
 > "If everything looks important, then nothing is important."
 >
-> ---Patrick Lencioni
+> <p>
+>   <span className="author">—Patrick Lencioni</span>
+> </p>
```

Comment:
I really wish there was a more semantic way to do this. It seems to be going away from the benefits of using MDX to get a small decorative benefit.

In terms of semantics, it would have to be a figcaption inside a figure wrapping the blockquote, which also seems suboptimal. Would it be possible to do it without the `p` tags at least?

Reply:
I overengineered this like a pro 😎  now the blockquote component detects whether the quote ends with em dash, and considers that to be the attribution, and applies the necessary styles, so now you can write in MDX normally without any JSX.